packages/query-core/src/mutationCache.ts

```diff
@@ -69,7 +69,7 @@ interface NotifyEventMutationUpdated extends NotifyEvent {
   action: Action<any, any, any, any>
 }
 
-type MutationCacheNotifyEvent =
+export type MutationCacheNotifyEvent =
```

Comment:
the index barrel on `query-core/index.ts` needs to re-export this to make it available to consumers. We don't magically re-export everything as that could leak internals.